package-lock.json

```diff
@@ -7365,12 +7368,14 @@
         "wrappy": {
           "version": "1.0.2",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
         },
         "yallist": {
           "version": "3.0.2",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
...so what are those? :sleepy:

Reply:
I don't know, but I had to be careful not to accidentally commit the version on my laptop where they got removed :roll_eyes: